src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/service/SnsService.kt

```diff
@@ -65,10 +65,26 @@ class SnsService(
   }
 }
 
+enum class WhatChanged {
+  BASIC_REPORT, // changes to a Report's basic information
+  ANYTHING, // anything in the report potentially changed
+  TYPE, // change to the report type
+  STATUS, // change to the report status
+  PRISONERS_INVOLVED,
+  STAFF_INVOLVED,
+  LOCATIONS,
+  EVIDENCE,
+  CORRECTION_REQUESTS,
+
+  // TODO: Just questions? Questions/answers? Endpoint not there yet
+  QUESTIONS,
```

Comment:
@ushkarev FYI: I've just added a `QUESTION` discriminant to this enum but I don't know how the endpoint(s) will be shaped so not sure if this is enough or needs tweaking.